### PR TITLE
RDM-5695: Increase timeout on ElasticSearch query

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
[RDM-5695] (https://tools.hmcts.net/jira/browse/RDM-5695).
  

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes 
[ ] No
```

### CODE REVIEW

File: charts/ccd-data-store-api/values.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ java:
     ELASTIC_SEARCH_BLACKLIST: query_string
     ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT: '%s_cases'
     ELASTIC_SEARCH_CASE_INDEX_TYPE: _doc
+    ELASTIC_SEARCH_TIMEOUT: 6000
```
Comment (mario-paniccia) [inline charts/ccd-data-store-api/values.yaml]: can you rename this to `ELASTIC_SEARCH_REQUEST_TIMEOUT` please

File: infrastructure/preview.tfvars [hunk 2]
```diff
@@ -3,3 +3,4 @@ idam_api_url = "https://idam-api.aat.platform.hmcts.net"
 document_management_valid_domain = "^https?://(?:api-gateway\\.preprod\\.dm\\.reform\\.hmcts\\.net|dm-store-aat\\.service\\.core-compute-aat\\.internal(?::\\d+)?)"
 default_print_url = "https://return-case-doc-ccd.nonprod.platform.hmcts.net/jurisdictions/:jid/case-types/:ctid/cases/:cid"
 frontend_url = "https://www-ccd.aat.platform.hmcts.net"
+elastic_search_timeout=8000
```
Comment (mario-paniccia) [inline infrastructure/preview.tfvars]: we can remove this one, we have the default value at 6 sec which is good for preview

File: charts/ccd-data-store-api/values.yaml (hunk 1 above)
Comment (mario-paniccia) [reply charts/ccd-data-store-api/values.yaml]: and let's set this to 10 sec please

Comment (Thor-tech-of-metal) [reply charts/ccd-data-store-api/values.yaml]: OK :)

File: infrastructure/variables.tf [hunk 3]
```diff
@@ -146,6 +146,10 @@ variable "elastic_search_enabled" {
   default = "false"
 }
 
+variable "elastic_search_timeout" {
+  default = "6000"
```
Comment (mario-paniccia) [inline infrastructure/variables.tf]: let's set to 10 sec please, and let's remove the explicit setting on prod.tfvars

File: infrastructure/preview.tfvars (hunk 2 above)
Comment (Thor-tech-of-metal) [reply infrastructure/preview.tfvars]: perfect :)

File: infrastructure/variables.tf (hunk 3 above)
Comment (Thor-tech-of-metal) [reply infrastructure/variables.tf]: DONE

File: dependency-check-suppressions.xml [hunk 4]
```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
+    <suppress>
+        <notes>Transitive dependency not used by our source. However it is needed for run time.</notes>
```
Comment (mario-paniccia) [inline dependency-check-suppressions.xml]: can you please check with Mutlu whether is ok to add this suppression? he's the specialist in this area :)

File: infrastructure/variables.tf [hunk 5]
```diff
@@ -146,6 +146,10 @@ variable "elastic_search_enabled" {
   default = "false"
 }
 
+variable "elastic_search_request_timeout" {
```
Comment (mario-paniccia) [inline infrastructure/variables.tf]: you have to modify the `main.tf` to assign this var value to an environment variable

Comment (Thor-tech-of-metal) [reply infrastructure/variables.tf]: DONE

**Pull request closed without merging**